app/controller.py

```diff
@@ -66,8 +66,10 @@ def changeToSaveAs(self):
     self.host.changeFocusTo(self.host.interactiveSaveAs)
 
   def doCommand(self, ch):
+    #Check the commandSet for the input with both its string and integer representation.
     self.savedCh = ch
-    cmd = self.commandSet.get(ch)
+    keyName = app.curses_util.cursesKeyName(ch)
```

Comment:
Actually, after some more thought, this is still hard coding in the values for these keys, so I don't think it will work across all platforms and terminals. I may have to try and see if it's possible to simulate these keys being pressed at the beginning of a session and store these values for future use (maybe save it as a file as well?). This should solve the problem of differing key values if python has a way to allow such a method.